src/Commands/Self/ConsoleCommand.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Self;
+
+use Pantheon\Terminus\Site\SiteAwareInterface;
+use Pantheon\Terminus\Site\SiteAwareTrait;
+use Pantheon\Terminus\Commands\TerminusCommand;
+
+class ConsoleCommand extends TerminusCommand implements SiteAwareInterface
+{
+    use SiteAwareTrait;
+    /**
+     * Initiates an interactive PHP console
+     *
+     * @command self:console
+     *
+     * @option string $site_env Site & environment to access as `$site` and (optional) `$env`
+     *
+     * @usage terminus self:console
+     *   Initiate an an interactive PHP console
+     * @usage terminus self:console mysite
+     *   Initiate an interactive PHP console with access to `$site` object
+     * @usage terminus self:console mysite.dev
+     *   Initiate an interactive PHP console with access to `$site` and `$env` objects
```

Comment:
The examples should use this tense for consistency:
`Initiates an interactive PHP console with access to the <env> environment of <site>.`